app/views/users/_photo.html.erb

```diff
@@ -29,6 +29,11 @@
     </p>
     <br class="d-md-none d-lg-inline" />
   </div>
+  <div style="float:right; margin-top:-58px;">
+    <%= link_to "/profile/photo/delete/"+current_user.uid.to_s, method: :delete, class: "btn btn-default" do %>
+      <i class="fa fa-trash"></i>
+    <% end %>
+  </div>
```

Comment:
Also is it possible to adjust the delete button so that it fits in the corner of the image upload box? Something like this should work

<img width="213" alt="Screenshot 2021-12-13 at 20 56 16" src="https://user-images.githubusercontent.com/63427719/145879320-226336c1-d4a0-4c39-be76-9a6659827be3.png">

Reply:
There should also be a confirmation window that opens that asks the user if they really want to delete the profile photo. This will help prevent accidental deletions. Thanks @sakshi-2412

Reply:
Yes sure! I'll update the changes ✨

Reply:
Thanks, these are great changes!